doc/releases/migration-guide-3.7.rst

```diff
@@ -170,6 +170,13 @@ Networking
   one IPv4 address / network interface, the netmask must be specified
   for each IPv4 address separately. (:github:`68419`)
 
+* Modified the `wifi connect` command to use key-value format for the arguments. In
+  previous implementation, we were identifying the option by option's position in argument string.
+  This makes it difficult to deal with optional arguments or extending the support
+  for other options. Having key-value format makes it easier to extend the options that
+  can be passed to connect command.
+  `wifi -h` will give more information about the usage of connect command.
```

Comment:
Could you add the link to this PR like in other bullet points, so
```
(:github:`70024`)
```

Reply:
Done.